### ISSUE TITLE
Export error with 2 instances of Reduxframework

### ISSUE THREAD

**Issue by trentb2** [reduxframework/redux-framework#1478]
First, love Reduxframework.
In a site I am developing we have Reduxframework inlcuded in a plugin for settings and also in a theme for options.
When export options from plugin all works fine. When export options for theme, get this error 'Invalid Secret for options use'. When I remove the lines creating error, the options export but upon viewing json file, it is the options for plugin and not theme.
Do I need to do something different when same site is using Reduxframework more than once in plugins or themes so that the correct export file is created? thanks.

**Comment by desaiuditd** [comment]
Hi,

I'm having exactly the same issue with two my plugins.

I've used embedded Redux in two plugins. Let's say Plugin A & Plugin B.

Have instantiated two class for them, say `Plugin_A_Settings` & `Plugin_B_Settings`. Both class is initiating `ReduxFramework` class respectively in `init_settings` method as suggested in documentation.

Now the issue I found out is here:

in `Redux_import_export` class, actions are defined like this.

``` php
add_action( "wp_ajax_redux_link_options", array( $this, "link_options" ) );
add_action( "wp_ajax_nopriv_redux_link_options", array( $this, "link_options" ) );

add_action( "wp_ajax_redux_download_options", array( $this, "download_options" ) );
add_action( "wp_ajax_nopriv_redux_download_options", array( $this, "download_options" ) );
```

Now the issue is, since I've initiated the class twice, these actions will be added twice. And while executing them, WordPress will only take the recent one into consideration. So if I want to download / copy options for Plugin A, it will always give me error, since it won't be able to match the secret. Because, while checking the secret it will check the secret of Plugin B since that time action from Plugin B object will be executed.

I've submitted a PR to solve this. I've tested it at my end and it works.
